### PR TITLE
Add README-zh_CN.md

### PR DESCRIPTION
**Pull request by richardo2016**
### Purpose

add official i18n README for zh-CN

### Changes

- add './README-zh_CN.md', and add entries both in it and `README.md`.

### Review Instructions

Just read `./README_zh-CN.md`

### CODE REVIEW

**Comment by richardo2016** [comment]
@rexwangcc @k-ye Hello, could you check if this PR necessary? 😂 I'm not sure README need to be translated.

File: README-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+文本档是 [Taichi](https://taichi.graphics) 的官方文档, 基于 [docusaurus](https://docusaurus.io/)
+
+---
+
+💡 **变更前须知**
+
+- 如果你想要**对 Taichi 的文档做任何变更**, 请移步[主仓库](https://github.com/taichi-dev/taichi/tree/master/docs)**而非在本仓库直接进行变更**! 该仓库仅仅是主仓库的文档的镜像版本, 专用于文档的编译和部署.
+- 如果你希望为太极的文档**贡献多语言翻译**, 请移步[**我们的翻译页面**](https://translate.taichi.graphics). 在 [help-us-translate](https://docs.taichi.graphics/help-us-translate) 页面有更多关于多语言工作的细节. **注意: 出于性能的原因, 我们并未支持对翻译内容的部署预览, 如果你只是做了部分翻译文本的变更, 通常不需要做整站的预览**.
+- 如果你希望 **变更任何无关版本的非文档类页面** (比如, `commnunity` 和 `help-us-translate` 页面), 在该仓库操作即可. 欢迎对本仓库的任何 PR!
+- 如果你希望**对网站做任何功能性变更**, 比如改变外观样式或者添加一个新的 React 组件, 在该仓库操作即可. 欢迎对本仓库的任何 PR!
```
Comment (rexwangcc) [inline README-zh_CN.md:20]:
```suggestion
- 如果你希望**对网站做任何功能性变更**, 比如改变外观样式或者添加一个新的 React 组件, 在本仓库操作即可. 欢迎对本仓库的任何 PR!
```

**Review by rexwangcc — APPROVED** [review]
LGTM, thank you very much for this PR, and yes, a README-zh_CN is really helpful, thanks! :octocat:

File: README-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+文本档是 [Taichi](https://taichi.graphics) 的官方文档, 基于 [docusaurus](https://docusaurus.io/)
+
+---
+
+💡 **变更前须知**
+
+- 如果你想要**对 Taichi 的文档做任何变更**, 请移步[主仓库](https://github.com/taichi-dev/taichi/tree/master/docs)**而非在本仓库直接进行变更**! 该仓库仅仅是主仓库的文档的镜像版本, 专用于文档的编译和部署.
+- 如果你希望为太极的文档**贡献多语言翻译**, 请移步[**我们的翻译页面**](https://translate.taichi.graphics). 在 [help-us-translate](https://docs.taichi.graphics/help-us-translate) 页面有更多关于多语言工作的细节. **注意: 出于性能的原因, 我们并未支持对翻译内容的部署预览, 如果你只是做了部分翻译文本的变更, 通常不需要做整站的预览**.
+- 如果你希望 **变更任何无关版本的非文档类页面** (比如, `commnunity` 和 `help-us-translate` 页面), 在该仓库操作即可. 欢迎对本仓库的任何 PR!
```
Comment (rexwangcc) [inline README-zh_CN.md:19]:
```suggestion
- 如果你希望 **变更任何无关版本的非文档类页面** (比如, `commnunity` 和 `help-us-translate` 页面), 在本仓库操作即可. 欢迎对本仓库的任何 PR!
```

**Pull request merged**